src/Playwright/Core/Page.cs

```diff
@@ -1357,6 +1357,9 @@ await Context.RecordIntoHarAsync(har, this, new()
                 Url = options?.Url,
                 UrlString = options?.UrlString,
                 UrlRegex = options?.UrlRegex,
+                Update = options.Update,
+                UpdateContent = options.UpdateContent,
+                UpdateMode = options.UpdateMode,
```

Comment:
`options` can be `null`

```suggestion
                Update = options?.Update,
                UpdateContent = options?.UpdateContent,
                UpdateMode = options?.UpdateMode,
```
(not sure if that compiles)